src/procedures/music-app/types.js

```diff
@@ -90,12 +107,27 @@ function registerTypes() {
     baseType: "Enum",
     baseParams: CHORDS,
   });
+
   types.defineType({
     name: "InstrumentNames",
     description: "List of available Instruments",
     baseType: "Enum",
     baseParams: NAMES,
   });
+
+  types.defineType({
+    name: "DrumOneShotTypes",
+    description: "List of available One-Shot Types",
```

Comment:
What is a "one-shot type"? A single drum beat?

Reply:
A one-shot means the sound is a single drum hit. So imagine we record a drummer hitting the snare one time. Thats what a one-shot is. I chose to categorize them this way to distinguish from a drum loop which would be more than just a single hit.